### PR TITLE
feat(catalog): Add kyverno

### PR DESCRIPTION
**Pull request by beiertu-mms**
See issue:
- https://github.com/kharf/flux-k8s-stack/issues/100

### CODE REVIEW

File: catalog/kyverno/kyverno-source.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (kharf) [inline catalog/kyverno/kyverno-source.yaml:1]: schon mal als hint: in den anderen packages hab ichs repositories.yaml genannt

File: catalog/kyverno/kyverno.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (kharf) [inline catalog/kyverno/kyverno.yaml:1]: und das hier releases.yaml

File: catalog/kyverno/releases.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      chart: kyverno
+      version: v2.6.5
+      sourceRef:
+        kind: HelmRepository
+        name: kyverno
+      interval: 1m
+  install:
+    crds: Create
+    remediation:
+      retries: 10
```
Comment (kharf) [inline catalog/kyverno/releases.yaml:20]: idk if we need that, havent specified that anywhere. User should decide how to handle failures imo

File: catalog/kyverno/repositories.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: kyverno
+  namespace: kyverno
+spec:
+  interval: 120m
```
Comment (kharf) [inline catalog/kyverno/repositories.yaml:8]: others are specified as  interval: 1h0m0s , I would make everything equal

File: catalog/kyverno/releases.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  chart:
+    spec:
+      chart: kyverno
+      version: v2.6.5
+      sourceRef:
+        kind: HelmRepository
+        name: kyverno
+      interval: 1m
+  install:
+    crds: Create
```
Comment (kharf) [inline catalog/kyverno/releases.yaml:18]: I would always use CreateReplace, so existing crds are upgraded

File: catalog/kyverno/releases.yaml (hunk 3 above)
Comment (beiertu-mms) [reply catalog/kyverno/releases.yaml]: it removed.

File: catalog/kyverno/repositories.yaml (hunk 4 above)
Comment (beiertu-mms) [reply catalog/kyverno/repositories.yaml]: the interval format are now aligned.

File: catalog/kyverno/releases.yaml (hunk 5 above)
Comment (beiertu-mms) [reply catalog/kyverno/releases.yaml]: it's now changed to `CreateReplace`.

**Review by kharf — APPROVED** [review]

**Pull request merged**